src/Aspire.Dashboard/wwwroot/css/app.css

```diff
@@ -151,6 +151,10 @@ fluent-data-grid-cell .long-inner-content {
     color: var(--neutral-foreground-rest);
 }
 
+ :root {
+     --error: #F8040A !important; /* the default error color (#BC2F32) is not accessible in dark theme */
```

Comment:
@vnbaaij I think the `!important` here is necessary because variables is included dynamically via the blazor module injection (indirectly via the `Microsoft.FluentUI.AspNetCore.Components.css`) so it comes last and we can't override with our own. Is there any way to avoid that, especially since there's so little in that file? I guess this will all change in 4.2 because of Javier's changes, right? So maybe best to leave this and deal with it after that?

Reply:
What changes are occurring in 4.2?

Reply:
This is what I was referring to: https://github.com/microsoft/fluentui-blazor/pull/1051
That moved some of the CSS into JS and adopted stylesheets. E.g. what was in variables.css is now here: https://github.com/microsoft/fluentui-blazor/blob/34b60e1bcd924dbef9ce3e24ae3de76ae3f21711/src/Core.Assets/src/index.ts#L15C1-L29C2

Reply:
Asked for clarity on when that will be released.